tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/Configs.scala

```diff
@@ -801,7 +810,30 @@ object Configs {
           checkPointPath
         )
       case SourceCategory.JANUS_GRAPH =>
-        JanusGraphSourceConfigEntry(SourceCategory.JANUS_GRAPH)
+        val name  = config.getString("name")
+        val label = config.getString("label")
+        val host  = config.getString("host")
+        val port  = config.getInt("port")
+        val checkPointPath =
+          if (config.hasPath("check_point")) Some(config.getString("check_point"))
+          else DEFAULT_CHECK_POINT_PATH
+        val parallel = if (config.hasPath("parallel")) config.getInt("parallel") else 1
+        val isEdge = {
+          if ((config.hasPath("vertex") || config.hasPath("vertex.field")) && (config.hasPath(
```

Comment:
Don't worry about the config conflict. Because other config entry don't check it. If the vertex is appeared, it's a tag.